.gitignore

```diff
@@ -0,0 +1,2 @@
+# Virtualenv
```

Comment:
Hey Khrisna, I don't think this is required because when we create a virtual environment with python, a `.gitignore` is created inside the root of the virtual environment with a `*` to ignore the entire environment. This way not everyone needs to have an environment name `venv`